src/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "outDir": "../lib"
     },
     "atom": {
-        "rewriteTsconfig": false
+        "rewriteTsconfig": true
```

Comment:
Also, when the grunt build is triggered, it will (I believe) change the files array back to its own style, and we'll have nasty back and forth diffs of tsconfig.json style.